bin/main.rb

```diff
@@ -1,67 +1,86 @@
 #!/usr/bin/env ruby
 
+require_relative '../lib/players'
+require_relative '../lib/game_status'
+
 class Board
   attr_accessor :board
```

Comment:
- [ ] Use attr_readers/attr_writers instead of attr_accessors everywhere you can. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))

- When I change it to be an attr_reader your game kept working normally so let's change it.